internal/sessiontest/session_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
```

Comment:
should we replace all usage of `io/ioutil` ?

Reply:
$ grep -crw ioutil * | grep -v ':0$'
internal/sessiontest/helper_dosession_test.go:3
internal/sessiontest/session_test.go:2
internal/keysharecore/testuser_test.go:2
internal/common/common.go:4
internal/test/testdata.go:2
internal/test/postgres.go:2
irma/cmd/sign.go:8
irma/cmd/keyshare-keygen.go:2
irma/cmd/keygen.go:3
irma/cmd/update.go:2
irmaclient/irmaclient_test.go:2
irmaclient/legacy.go:2
irmaconfig.go:2
irmago_test.go:2
keyring.go:2
requests.go:2
schemes.go:9
server/keyshare/keyshareserver/conf.go:2
server/api.go:2
transport.go:2

Reply:
Fixed in #303 to prevent pull request pollution.